tests/system/libraries/SystemTestSpy/speechSpyGlobalPlugin.py

```diff
@@ -197,13 +198,30 @@ def wait_for_specific_speech(
 			errorMessage=None
 		)
 		if not success:
+			if self._popup_stole_focus():
+				return self.wait_for_specific_speech(speech, afterIndex, maxWaitSeconds)

```

Comment:
I actually don't think I saw any tests where this window being created mid system test was likely. Certainly there were situations where the popup had focus, but I doubt it was due to the popup being created mid test. Instead I suspect it was a result of NVDA "restoring focus" over the top of chrome due to a race condition during startup. I expect that the popup is first created much earlier, during the build.

Perhaps a comment in here (or the function) to say that this is to collect data on whether this is happening. And may be removed if we determine that it is not. An alternative approach may be to have an "allow list" of windows / processes that are allowed to gain focus after the test is started, anything else is a failure with the process name reported.

Reply:
This test where we sleep for 10s failed due to a docker popup: https://ci.appveyor.com/project/NVAccess/nvda/builds/38672328/artifacts. 

@michaelDCurran also noted that we probably shouldn't be killing processes that developers may be running when running system tests locally. I'm going to remove this code in favour of the quick sleep test and we can follow up to address the Docker issue later.